packages/docs/docs/how-tos/get-status.md

```diff
@@ -0,0 +1,86 @@
+
+Please note that the status reported by the Web3.Storage service may not always be in sync with the actual status as reported by the IPFS cluster. The Web3.Storage service checks if `Failed` pins have been found and pinned by the cluster once per week for up to one month after the initial upload.
+
+If you find that your file is retrievable but the pin status still says `Failed`, please [file an issue][new-issue] with the relevant CIDs and we'll take a look.
+
+## Storage providers
+
+The **Storage Providers** column on the file listing page shows information about the [Filecoin storage providers][fil-docs-storage-providers] that have committed to long-term storage deals for the uploaded data.
+
+When data is first uploaded, the Storage Providers column will show a status of `Queuing`. This indicates that the Web3.Storage service has the data and is aggregating it into a format that can be used to make a storage deal.
+
+Once enough data has been aggregated, the Web3.Storage service will propose several Filecoin [storage deals][fil-docs-deals] with different Filecoin storage providers. These will initially be shown in a `Pending` status, while the provider accepts the deal and posts its commitment to the Filecoin blockchain.
+
+When using the HTTP API or client library to check the upload status, information about Filecoin storage is provided in the `deals` array field within the status response object. The `deals` array will be empty while the upload is in the `Queuing` state.
+
+Once storage deals have been proposed, the `deals` array will contain JSON objects describing each deal. Each object has a `status` field, which will have a value of `"queued"` when the deal is pending. When the deal has been confirmed and published to the Filecoin blockchain, the `status` field will have a value of `"active"`.
+
+Here's an example JSON object that describes an active storage deal:
+
+```json
+{
+    "status": "active",
+    "lastChanged": "2022-03-13T04:17:47.969874+00:00",
+    "chainDealID": 4486352,
+    "datamodelSelector": "Links/197/Hash/Links/22/Hash/Links/0/Hash",
+    "statusText": "containing sector active as of 2022-03-13 09:18:00 at epoch 1628556",
+    "dealActivation": "2022-03-14T13:19:00+00:00",
+    "dealExpiration": "2023-08-06T13:19:00+00:00",
+    "miner": "f0840770",
+    "pieceCid": "baga6ea4seaqcfs2mrjyaqnwgdheb4ny76i6yaeeklcylqijaha3kpcwmutnb6ma",
+    "batchRootCid": "bafybeig2qsdu6nfmmaxlpnrpcz63jzyb6bzc4ngllv4mcj7q66ei4rq72y"
+}
```

Comment:
This is the NFT.Storage deal schema please check https://api.web3.storage/status/bafybeiauyddeo2axgargy56kwxirquxaxso3nobtjtjvoqu552oqciudrm and fix.

Reply:
It would be great to define these fields: https://github.com/web3-storage/web3.storage/blob/cd1889177abe20d7fd0fa981e2c419a79725b64d/packages/client/src/lib/interface.ts#L193-L230

Reply:
Corrected the JSON example and added a schema table.